src/apis/setDatePickerLocale.js

```diff
@@ -0,0 +1,17 @@
+import core from 'core';
+
+export default function(i18nextPromise) {
+  i18nextPromise.then(t => {
```

Comment:
This file should be put under the `helpers` folder as this is not an API that's attached to `window.readerControl`.